### PR TITLE
Consistent names

### PR DESCRIPTION
**Pull request by jbothma**
unifying all cubes to refer to item as item is important for automatically the items as in the return forms. Also time to clean up the names now we understand their usage better before they get used a lot and we break more things by changing them later.

@longhotsummer FYI

### CODE REVIEW

File: models/aged_creditor.json [hunk 1]
```diff
@@ -7,21 +7,21 @@
             "attributes": {
                 "code": {
                     "column": "demarcation_code",
-                    "label": "Code",
+                    "label": "Demarcation Code",
                     "type": "string",
                     "description": "Demarcation Code desc"
```
Comment (longhotsummer) [inline models/aged_creditor.json]: While you're at it, why not make this:

Municipality code assigned by the Municipal Demarcation Board

File: models/aged_creditor.json [hunk 2]
```diff
@@ -7,21 +7,21 @@
...
                 },
                 "desc": {
-                    "column": "demarcation_desc",
-                    "label": "Description",
+                    "column": "demarcation_label",
+                    "label": "Name",
                     "type": "string",
                     "description": "some description"
```
Comment (longhotsummer) [inline models/aged_creditor.json]: Municipality name

File: models/aged_creditor.json [hunk 3]
```diff
@@ -7,21 +7,21 @@
...
                 }
             },
             "key_attribute": "code",
             "label_attribute": "desc",
             "description": "Demarcation of location or some such",
```
Comment (longhotsummer) [inline models/aged_creditor.json]: Municipality demarcation details.

**Comment by longhotsummer** [comment]
Let's take this chance to fill in some of the other placeholders, too.

File: models/aged_creditor.json (hunk 1 above)
Comment (jbothma) [inline models/aged_creditor.json]: Sure. I like. For codes I think we can say where they're defined - e.g. I'll say which codes are the treasury's. This one's the demarcation board's etc.

File: models/aged_creditor.json (hunk 2 above)
Comment (jbothma) [inline models/aged_creditor.json]: I think this gets redundant when the dimension label is municipality and the attribute label is Name. I think it's best leaving the description blank in such cases

Comment (longhotsummer) [inline models/aged_creditor.json]: I'm happy with that.

**Comment by jbothma** [comment]
This is so freaking tedious and repetitive. Only reason I haven't templated this is because this shouldn't need a lot of changes once it's properly released

**Pull request merged**